### PR TITLE
Changes in the footer

### PR DESCRIPTION
**Pull request by netdesignr**
Hi @sdwilkes , can you please kindly review this pull request?

1. Added opening times inside footer
2. Added follow us icons inside footer
3. Adapted the grid system and changed the position of the columns for mobile and devices
4. Added footer stylesheet
5. Tested in Chrome, Safari, and Firefox

Thank you!

### CODE REVIEW

File: inc/content/footer-content.php [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                    The National Archives<br/>
+                    Kew, Richmond, Surrey,<br/>
+                    TW9 4DU
+                </address>
+                Tel: <a class="tel" href="tel:+4402088763444">+44 (0) 20 8876 3444</a>
+            </div>
+            <div class="footer-col col-xs-12 col-sm-6 col-md-12 col-lg-12">
+                <h4>Standard opening times</h4>
+                <div class="opening-times">
+                    <table>
```
Comment (sdwilkes) [inline inc/content/footer-content.php]: Can we use a display:table instead?

File: inc/content/footer-content.php [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    </div>
+    <div class="footer-col col-xs-12 col-sm-12 col-md-3">
+        <div class="row">
+            <div class="footer-col col-xs-12 col-sm-6 col-md-12">
+                <h4>Follow us</h4>
+                <div class="follow-us">
+                    <a href="https://twitter.com/@UKNatArchives" title="Follow us on Twitter">
+                        <img src="<?php echo get_stylesheet_directory_uri() ?>/img/social/twitter.png"
+                             alt="Twitter logo"></a>
+                    <a href="https://www.youtube.com/c/TheNationalArchivesUK">
```
Comment (sdwilkes) [inline inc/content/footer-content.php]: Please add title tags for the other social media links

File: css/sass/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    display:inline-block;
+    img {
+      background-color: #008484;
+      border-radius: 50%;
+      padding: 9px;
+      margin: 0 0.2em 0.3em 0;
+      width:46px;
+    }
+  }
+}
```
Comment (sdwilkes) [inline css/sass/_footer.scss]: Can we add a hover state to the follow-us class?

File: inc/content/footer-content.php (hunk 2 above)
Comment (netdesignr) [inline inc/content/footer-content.php]: Fixed

File: inc/content/footer-content.php (hunk 1 above)
Comment (netdesignr) [inline inc/content/footer-content.php]: Removed the table and using only divs

File: css/sass/_footer.scss (hunk 3 above)
Comment (netdesignr) [inline css/sass/_footer.scss]: Done - Please have a look :)

**Pull request merged**